e2e_playwright/st_arrow_vega_lite_chart_test.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from playwright.sync_api import Page, expect
+
+from e2e_playwright.conftest import ImageCompareFunction
+
+
+def test_arrow_vega_lite_chart(themed_app: Page):
+    """Tests that it displays charts on the DOM"""
+    arrow_vega_lite_charts = themed_app.locator("[data-testid=stArrowVegaLiteChart]")
+    expect(arrow_vega_lite_charts).to_have_count(13)
+
+    for idx in range(13):
+        chart = arrow_vega_lite_charts.nth(idx)
+        canvas = chart.locator("canvas").nth(0)
+        expect(canvas).to_be_visible()
+        expect(canvas).to_have_class("marks")
+
+
+def test_arrow_vega_lite_chart_sets_chart_width(themed_app: Page):
+    """Tests that it sets the correct chart width"""
+    arrow_vega_lite_charts = themed_app.locator("[data-testid=stArrowVegaLiteChart]")
```

Comment:
nit: `arrow_vega_lite_charts = themed_app.locator("[data-testid=stArrowVegaLiteChart]")` could be
```suggestion
    arrow_vega_lite_charts = themed_app.get_by_test_id("stArrowVegaLiteChart")")
```